json-lint/schemas/inventory/adservers/adsense.json

```diff
@@ -61,16 +61,32 @@
       },
       "auto-format": { "const" : "rspv" },
       "width": {
-        "type": ["integer","string"],
-        "title": "String of digits ( if not set, default is 300 )",
-        "description": "example: 300",
-        "pattern": "^(\\$\\{.+\\})$"  
+        "oneOf": [
```

Comment:
lets point here to /adservers/types/dimensions.json with a $ref

Reply:
Fixed